README.md

```diff
@@ -1 +1,72 @@
-# .github
\ No newline at end of file
+
+## Repositories: 
+- Service
+- Purpose
+- Language
+- External Integration
+- Database
+- Infrastructure
+- Main contributor
+
+### 1. ubiquibot/conversation-rewards
+- Service: Conversation Reward Module
+- Purpuse: This is intended to be the proper implementation of comment incentives, based on our learnings from the first go-around.
+- Language: TypeScript
+- External Integration: Github Action, Github API
+- Database: None
+- Main contributor: @pavlovcik
```

Comment:
```suggestion
```

I'm not sure if main contributor is useful to add because this is very dynamic and I'm not really supposed to be coding these days (I am focusing on fundraising and hiring since 1 March.)